tests/ballerina-test-utils/src/main/java/org/ballerinalang/test/BRunUtil.java

```diff
@@ -302,18 +307,35 @@ public static ExitDetails run(CompileResult compileResult, List<String> javaOpts
             classPath.append(File.pathSeparator).append(jarLibrary.path());
         }
 
-        try {
-            final List<String> actualArgs = new ArrayList<>();
-            int index = 0;
-            actualArgs.add(index++, "java");
-            for (String javaOpt : javaOpts) {
-                actualArgs.add(index++, javaOpt);
+        final List<String> actualArgs = new ArrayList<>();
+        int index = 0;
+        actualArgs.add(index++, "java");
+        for (String javaOpt : javaOpts) {
+            actualArgs.add(index++, javaOpt);
+        }
+        actualArgs.add(index++, "-cp");
+
+        String classPathString = System.getProperty("java.class.path") + classPath;
+        // Create an argument file for windows to mitigate the long classpath issue.
```

Comment:
```suggestion
        // Create an argument file for the Windows OS to mitigate the long classpath issue.
```

Reply:
Isn't it informative enough to use `Windows` here?

Reply:
It is enough.